src/connectors/cacheDb/cacheDb.connector.js

```diff
@@ -121,6 +149,65 @@ const updateStatusInCache = async (fsa_rn, property, value) => {
   }
 };
 
+const findAllOutstandingSavesToTempStore = async (
+  cachedRegistrations,
+  limit = 100
+) => {
+  return await cachedRegistrations
+    .find({
+      $or: [
+        { "status.registration.complete": { $eq: false } },
+        { "status.registration": { $exists: false } }
+      ]
+    })
+    .sort({ reg_submission_date: 1 })
+    .limit(limit);
+};
+
+const findAllOutstandingNotificationsRegistrations = async (
+  cachedRegistrations,
+  limit = 100
+) => {
+  return await cachedRegistrations
+    .find({
+      $or: [
+        {
+          "status.notifications": {
+            $elemMatch: { sent: { $ne: true } }
+          }
+        },
+        { "status.notifications": { $exists: false } }
+      ]
+    })
+    .sort({ reg_submission_date: 1 })
+    .limit(limit);
+};
+
+const findOutstandingTascomiRegistrationsFsaIds = async (
+  cachedRegistrations,
+  limit = 100
+) => {
+  return await cachedRegistrations
+    .find(
+      {
+        $or: [
+          { "status.tascomi.complete": { $eq: false } },
+          { "status.tascomi": { $exists: false } }
```

Comment:
thsi definitely needs amending: if there is no tascomi section, they shouldn't be sending to tascomi anyway.

Reply:
I was thinking - so we have changed this so that the tascomi stuff isnt even attempted on save in becache. So this means i am thinking that the tascomi status is never run for new items. Therefore I think this is necessary now on reflection.

Reply:
Then I think we need to assign the empty status when the back end cache record is created, as it was before - based on whether there is tascomi auth present. Otherwise, we risk breaching Cosmos database throughput limits (these are small, cheap accounts) - I've encountered this problem when trying to update mnay records on tests system.

Reply:
cool beans - will put back in. I understand re the cheap accounts - incidentally would the consolidation of mongos be more cost effective?

Reply:
ST ANW - will leave out for now as not necessary on reflection.